CONTRIBUTING.md

```diff
@@ -1,5 +1,5 @@
 # Contributing guidelines
 
-Please follow the same [contributing guide](https://github.com/kubevirt/community/blob/master/contributers/contributing.md) as you would follow for any other pull request made to Kubevirt.
+Please follow the same [contributing guide](https://github.com/kubevirt/community/blob/master/contributors/contributing.md) as you would follow for any other pull request made to Kubevirt.
```

Comment:
What's the point of having a reference to "Page under construction. Check back soon."?